assets/js/translator.js

```diff
@@ -736,6 +736,10 @@ function downloadBrowserWarn() {
     }
 }
 function detectLanguage() {
+    if($('#originalText').val().length === 0) {
```

Comment:
Extract `$('#originalText').val()` into a variable and replace other instances, perhaps `var originalText` since it's used later in this function as well.